README.md

```diff
@@ -377,6 +377,29 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
```

Comment:
I suggest adding an OAuth section to the readme to provide some details about this feature. See the following examples:
* https://github.com/okta/okta-sdk-php#oauth-20
* https://github.com/okta/okta-sdk-dotnet#oauth-20

Reply:
Noted. @bdemers Would it be fine if I create a new PR for this suggested README change before we officially release it?

Reply:
Discussed this with @bdemers and agreed on this idea to create a new PR to push the entire changes to README. Will revert this change to README before merging this PR.

Reply:
Here is the link to the PR for README changes: https://github.com/okta/okta-sdk-java/pull/366